client/index.jsx

```diff
@@ -27,37 +34,70 @@ class App extends React.Component {
         .then((data) => {
           this.setState({
             reviews: data.data,
+            filteredReviews: data.data,
           });
         })
         .catch((err) => console.error(err));
     }
   }
 
+  filterReviews(key, value) {
+    const { reviews } = this.state;
+    const { filterToggle } = this.state;
+    if (filterToggle.starRaiting === value) {
+      this.setState({
+        reviewDisplayCount: 3,
+        filteredReviews: reviews,
+        filterToggle: {
+          starRaiting: 0,
```

Comment:
Hey Matt: why would you be setting filterToggle.starRating to be zero here? Wouldn't you want to keep it at the current value?

Reply:
the .starRaiting is a way for me to toggle the onClick event for star filtering. I can only have one star-rating filter active at a time, and as I have 5 different states to track, I can't simply use a boolean value. In this instance, zero is my false value, and any number between 1-5 is true - for toggling the filter on or off. Hope that makes sense!